proto/allocation/allocation.proto

```diff
@@ -196,3 +202,21 @@ message Priority {
   }
   Order order = 3;
 }
+
+// CounterAction is an optional action that can be performed on a Counter at allocation.
+// Action: "Increment" or "Decrement" the Counter's Count (optional). Must also define the Amount.
+// Amount: The amount to increment or decrement the Count (optional). Must be a positive integer.
+// Capacity: Update the maximum capacity of the Counter to this number (optional). Min 0, Max int64.
+message CounterAction {
+  google.protobuf.StringValue action = 1;
+	google.protobuf.Int64Value amount = 2;
+	google.protobuf.Int64Value capacity = 3;
```

Comment:
We've used `int64` in the other proto defintinions in this file - but we're using `google.protobuf.Int64Value` here?

I do see we also use it here:

https://github.com/googleforgames/agones/blob/d2189ca7765e8089b1664242dae3a821f5b96a1d/proto/sdk/alpha/alpha.proto#L231-L245

But only noticed the difference now, so curious what the difference is?

Does this provide a nullable value on the generated code side?

Reply:
We went with pointers so that the field could be nil. Otherwise int64 defaults to 0 which in this case is a valid input value.

Reply:
Gotcha! Make sense - thanks for confirming!